Traceback (most recent call last):
  File "pseudo_mini_batch_range_arxiv_sage.py", line 3, in <module>
    import dgl
  File "/home/cc/.local/lib/python3.6/site-packages/dgl/__init__.py", line 13, in <module>
    from .backend import load_backend, backend_name
  File "/home/cc/.local/lib/python3.6/site-packages/dgl/backend/__init__.py", line 107, in <module>
    load_backend(get_preferred_backend())
  File "/home/cc/.local/lib/python3.6/site-packages/dgl/backend/__init__.py", line 30, in load_backend
    import torch
  File "/home/cc/.local/lib/python3.6/site-packages/torch/__init__.py", line 190, in <module>
    from torch._C import *
ImportError: numpy.core.multiarray failed to import
